src/Platforms/AbstractPlatform.php

```diff
@@ -185,6 +185,19 @@ private function initializeAllDoctrineTypeMappings()
         }
     }
 
+    /**
+     * Returns the SQL snippet used to declare a column that can
+     * store characters in the ASCII character set
+     *
+     * @param mixed[] $column
+     *
+     * @return string
+     */
+    public function getAsciiStringTypeDeclarationSQL(array $column)
```

Comment:
Any reason not to use a type declartion here?

```suggestion
    public function getAsciiStringTypeDeclarationSQL(array $column): string
```

Reply:
> Any reason not to use a type declaration here?

I believe it's because all other methods don't have it and it's not enforced by the standard. Although it's a good thing to use them for new code.